tservice/src/seal.rs

```diff
@@ -28,7 +42,10 @@ impl<T: AsRef<[u8]> + core::default::Default> SealedBuilder<T> {
     pub fn new(data: T) -> Self {
         Self {
             data,
-            ..Default::default()
+            policy: DEFAULT_KEY_POLICY_FOR_SEAL,
```

Comment:
I'm reluctant to default `KeyPolicy` to `MRSIGNER` everywhere else.  It's used in the `KeyRequest` and I wouldn't want people to assume that `MRSIGNER` is the preferred key method.  It looks like `sgx_get_key()` isn't currently used by mobilecoin repo, however since this crate is meant to be more generic I think it's better to avoid the default based on only the mobilecoin repo needs.